### DISCUSSION TITLE
작동은 안되고 그냥 꺼집니다...

### DISCUSSION THREAD

**Discussion by ghost** [storycraft/node-kakao#461] [Q&A]
카카오톡에 로그인 했다고 뜨는데 그냥 꺼지네요
![loco](https://user-images.githubusercontent.com/75028791/123808072-64080380-d92b-11eb-89ce-866b78f77046.png)

**Comment by raravel** [comment]
코드나 에러 내용도 없이 꺼진다고 하면 아무도 이유를 모릅니다.

**Comment by generalxd** [comment]
이건뭐 ;

**Reply by ghost** [reply]
고쳤습니다
